site/DonateForm.tsx

```diff
@@ -29,11 +29,25 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome/index.js"
 import { faArrowRight, faInfoCircle } from "@fortawesome/free-solid-svg-icons"
 import { SiteAnalytics } from "./SiteAnalytics.js"
 
-const ONETIME_DONATION_AMOUNTS = [20, 50, 100, 500, 1000]
-const MONTHLY_DONATION_AMOUNTS = [5, 10, 25, 50, 100]
+const ONETIME_DONATION_AMOUNTS = [20, 50, 100, 500, 1000] as const
+const MONTHLY_DONATION_AMOUNTS = [5, 10, 25, 50, 100] as const
+const ANNUAL_DONATION_AMOUNTS = [60, 120, 300, 600, 1200] as const
 
 const ONETIME_DEFAULT_INDEX = 2
 const MONTHLY_DEFAULT_INDEX = 2
+const ANNUAL_DEFAULT_INDEX = 2
```

Comment:
nit: I find this whole default index thing a bit over-engineered. We're clearly aiming for the psychological middle choice here across all intervals, and it seems unlikely that we'll ever need that level of granularity.